llmfoundry/command_utils/eval.py

```diff
@@ -127,7 +112,7 @@ def evaluate_model(
     fsdp_config = parallelism_config.get(
         'fsdp_config',
         None,
-    ) if parallelism_config else fsdp_config
```

Comment:
below when making the trainer object, i think we should just pass along `parallelism_config` instead of making it again